hack/smoketest/main.go

```diff
@@ -23,6 +23,7 @@ import (
 	"path/filepath"
 	"time"
 
+	log "github.com/sirupsen/logrus"
```

Comment:
nit: Do we need the logging here on top of the `fmt.Print` statements? If we remove this (or at least use logr) we'd drop the the new dependency added to go.mod